src/chromium/__tests__/ext-background.test.js

```diff
@@ -56,14 +56,19 @@ test('sets the post-uninstall URL', () => {
     .toHaveBeenCalledWith('https://tab.gladly.io/newtab/uninstalled/')
 })
 
-test('gracefully handles errors with setting the post-uninstall URL', () => {
```

Comment:
It looks like you accidentally replaced an existing test. Please re-add that test.